### ISSUE TITLE
Service logs command fails on long lines

### ISSUE THREAD

**Issue by marioevz** [kurtosis-tech/kurtosis#1188]
### What's your CLI version?

0.82.11

### Description & steps to reproduce

When running the `eth2-package` with the following json config launch file:
```json
{
  "participants": [
    {
      "el_client_type": "geth",
      "el_client_image": "ethpandaops/geth:lightclient-devnet-8-edf4ab3",
      "cl_client_type": "teku",
      "cl_client_image": "consensys/teku:develop",
      "count": 2
    }
  ],
  "network_params": {
    "capella_fork_epoch": 1,
    "deneb_fork_epoch": 2
  },
  "launch_additional_services": true,
  "global_client_log_level": "debug",
  "snooper_enabled": true,
  "mev_type": "mock"
}
```
And then doing:
```bash
kurtosis service logs eth2 cl-2-teku-geth
```
The command's output is cropped with the following error at the end:
```
ERRO[2023-08-29T16:45:27Z] An error occurred receiving user service logs stream for user services 'map[d8ce7cc991fd45ef99e5a9dac2f1db83:true]' in enclave 'eth2'. Error:
rpc error: code = Unknown desc = An error occurred streaming user service logs
 --- at /home/circleci/project/engine/server/engine/server/engine_connect_server_service.go:220 (EngineConnectServerService.GetServiceLogs) ---
Caused by: An error occurred parsing the json logs file for service 'd8ce7cc991fd45ef99e5a9dac2f1db83' in enclave 'd43157f6df504fcb94bce098af8257c8' at the following path: /var/log/kurtosis/d43157f6df504fcb94bce098af8257c8/d8ce7cc991fd45ef99e5a9dac2f1db83.json.
 --- at /home/circleci/project/engine/server/engine/centralized_logs/client_implementations/persistent_volume/persistent_volume_logs_database_client.go:186 (streamServiceLogLines) ---
Caused by: unexpected end of JSON input
```
On previous versions (I had been using 0.81.9) the logs showed just fine and this line where the logs output is failing is a very long line (perhaps that's the reason?).

### Desired behavior

`kurtosis service logs eth2 cl-2-teku-geth` should show the full log without issues.

### What is the severity of this bug?

Papercut; this bug is frustrating, but I have a workaround.

**Comment by leeederek** [comment]
Hey @marioevz - thanks for submitting this bug report. I've been able to reproduce this issue on my side. Will continue to investigate and let you know what we find. Thank you!

**Comment by leeederek** [comment]
Hey @marioevz this should be fixed as of 0.82.12 as per: https://github.com/kurtosis-tech/kurtosis/pull/1175

Can you give it a try and let me know if it solves the issue on your end?

**Comment by marioevz** [comment]
Hey @leeederek I reran the local testnet and now there is no error but the logs of all services seem to be trimmed by default to the 200th line for some reason, this is on version `0.82.14` and had not happened before.

**Comment by leeederek** [comment]
Hey @marioevz thanks for the feedback. I've confirmed that we do indeed truncate the logs in the terminal at 200 lines. While we can easily bump this, I am curious about your workflow in case there is a cleaner way to help you achieve what you're trying to do.

Can you share with me how you're consuming or using the logs? depending on what your needs are, I have a few options that you might find useful:
1. If you're trying to get a stream of logs, I recommend you try [`kurtosis service logs $enclave_name $service_name -f`](https://docs.kurtosis.com/service-logs) to tail or follow the logs. Alternatively, if you prefer, you can try `http://localhost:9711/enclaves` to see the logs in a web-browser format. Make sure you're on Kurtosis `0.82.15` though - we just found a bug in `0.82.14` that we fixed in `0.82.15`.
2. If you're ok with just grabbing a snapshot of the logs up to a certain point in time (after an event or a change you made), you can do a [`kurtosis dump`](https://docs.kurtosis.com/dump) which will export all the logs relating to your enclaves, your kurtosis engine instance, and all services within your enclaves into your working directory for you to search through and investigate.
3. If you're looking for a keyword in the logs, I would recommend the various matching flags we offer in `kurtosis service logs`, which includes regex matching!

**Comment by leeederek** [comment]
Spoke to Mario on Discord to resolve/wrap up the topic - closing this out!
